tests/integration/payment-gateway/SV_WC_Payment_Gateway_Payment_Tokens_Handler_Test.php

```diff
@@ -26,6 +26,81 @@ protected function _after() {
 	}
 
 
+	/**
+	 * @see Framework\SV_WC_Payment_Gateway_Payment_Tokens_Handler::delete_token()
+	 */
+	public function test_delete_token() {
```

Comment:
it doesn't look like this test contains any assertions

Reply:
The `\Codeception\Stub::make(...)` part causes the test to fail if `delete()` is **not** called on the token mock exactly once before the test ends.

However, I think I can write this test using an approach similar to the one used `test_delete_token_marks_another_token_as_deafult` which kind of aligns better with the integration nature of these tests. Then I can move some of those other asserts to this one as you suggested in another comment.